packaging/src/main/assembly/src.xml

```diff
@@ -41,6 +41,12 @@
         <exclude>**/.project</exclude>
         <exclude>**/.settings/**</exclude>
         <exclude>**/thirdparty/**</exclude>
+        <exclude>standalone-metastore/metastore-common/src/gen/version/**</exclude>
+        <exclude>standalone-metastore/metastore-server/src/gen/**</exclude>
+        <exclude>common/src/gen/**</exclude>
+        <exclude>kafka-handler/src/test/gen/**</exclude>
+        <exclude>conf/hive-default.xml.template</exclude>
+        <exclude>ql/dependency-reduced-pom.xml</exclude>
```

Comment:
Can we make the exclusions (or some of them) more generic to avoid including again similar things in the future? For instance exclude all kind of dependency-reduced-pom.xml files and not only under ql.